cmd/snap/cmd_snap_op.go

```diff
@@ -342,11 +342,13 @@ func (mx *channelMixin) asksForChannel() bool {
 type modeMixin struct {
 	DevMode  bool `long:"devmode"`
 	JailMode bool `long:"jailmode"`
+	Classic  bool `long:"classic"`
 }
 
 var modeDescs = mixinDescs{
 	"devmode":  i18n.G("Request non-enforcing security"),
 	"jailmode": i18n.G("Override a snap's request for non-enforcing security"),
+	"classic":  i18n.G("Consent to use classic confinement"),
```

Comment:
Yeah let's get some better wording for this. I specifically picked _consent_ as it implies "agreement to something potentially dangerous" and I think the connotation is just right but that's just a personal view.